packages/uswds-core/src/styles/mixins/general/table.scss

```diff
@@ -356,6 +362,46 @@ $table-unsorted-icon-color: get-color-token-from-bg(
   @include table-stacked-header-styles;
 }
 
+@mixin usa-table--sticky-header {
+  border: 1px solid color($theme-table-border-color);
+  border-collapse: separate;
+
+  td,
+  th {
+    border-left: none;
+    border-top: none;
+
+    &:last-child {
+      border-right: none;
+    }
+  }
+
+  tbody tr:last-child {
+    td,
+    th {
+      border-bottom: none;
+    }
+  }
+
+  thead {
+    position: sticky;
+    top: $theme-table-sticky-top-offset;
```

Comment:
> **Warning**
> This offset is useful for when there is another element that already sticks to the top of the page, like a sticky header navigation. However, I suspect that most users will need to write custom rules if their sticky elements changes at different breakpoints (For example, our sticky header navigation goes away in the mobile view). Do we want to keep this setting with the understanding that additional style overrides might often be necessary?

Reply:
I feel like we should keep it. Like most of our work, we provide the tools, but it's up to the users to implement them in a way that works for them.